doc/attr.xml

```diff
@@ -1461,6 +1461,35 @@ gap> HamiltonianPath(g);
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="DigraphCore">
+  <ManSection>
+    <Oper Name = "DigraphCore" Arg = "D"/>
+    <Returns>A list of positive integers.</Returns>
+    <Description>
+      If <A>D</A> is a digraph, then <C>DigraphCore</C> returns a list of vertices
```

Comment:
What you've implemented is the list of vertex labels corresponding to the core. I think you actually want to be returning the list of vertices, which is what the documentation suggestions. Vertex labels have no requirements on them, they're not even required to be distinct, which makes them not very useful for being things like the output of computations. For instance, look at this:
```
gap> gr := CycleDigraph(4);;
gap> SetDigraphVertexLabels(gr, [fail, fail, fail, fail]);
gap> DigraphCore(gr);
[ fail, fail, fail, fail ]
```
Not very helpful! There might be a good reason to use non-unique vertex labels, but if you do so this function isn't very useful.